source/transactions/tests/error-labels.yml

```diff
@@ -670,6 +673,68 @@ tests:
         data:
           - _id: 1
 
+  - description: do not add RetryableWriteError label to writeConcernError ShutdownInProgress that occurs within transaction
+
+    failPoint:
+      configureFailPoint: failCommand
+      mode: { times: 1 }
+      data:
+          failCommands: ["insert"]
+          writeConcernError:
+            code: 91
+            errmsg: Replication is being shut down
+
+    operations:
+      - name: startTransaction
+        object: session0
+        arguments:
+          options:
+            writeConcern:
```

Comment:
This configures the transaction to use write concern majority, but the `abortTransaction` expectation below expects `writeConcern: `. Per https://github.com/mongodb/specifications/blob/master/source/transactions/transactions.rst#writeconcern, I think the expectation should include majority. Alternatively, we could just remove the `writeconcern` field from the expectation. That seems like something that should be tested by the options tests, not this.

Reply:
Yup, agreed.